volatility3/framework/__init__.py

```diff
@@ -5,23 +5,8 @@
 # Check the python version to ensure it's suitable
 import glob
 import sys
+import volatility3.framework.check_python_version  # noqa: F401
```

Comment:
it would probably work but imo the noqa is more readable, if i just saw that i'd ask myself, why import that as that

Reply:
Yeah, I take your point.  Perhaps (in either case) we should add a comment explaining that the import has side-effects, either to run some code directly, or to ensure classes are registered so they show up as part of the plugin mechanism?  I just hate seeing code littered with exceptions because it makes people think the error checker can just be turned off rather than the problem fixed.  The fewer of those in the codebase, the better in my view (which is why a legit, but odd looking line is better than one we explicitly turn the error checker off for).

Reply:
usually i'd opt for no side effects at all (and enabling all linting rules and ignoring single ones) but this doesn't really seem possible here (size of the codebase, the way its built right now etc.) i'm not quite sure what to do here.

additionally the `F401` isn't fixed by a
```py
import volatility3.framework.check_python_version as check_python_version
```

if i understand the rule correctly (and based on some testing) its `from .. import foo as foo` that counts as "redundant import"

Reply:
(seems like i was wrong)

Reply:
fixed (using redundant alias name) here: https://github.com/volatilityfoundation/volatility3/pull/1364/commits/f2771bce87e80c7a988247f855e5afa117753654